deployment/ansible/deploy.yml

```diff
@@ -13,18 +13,76 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-- name: bootstrap DCL nodes
+- name: bootstrap all nodes
   hosts: all
   become: true
   roles:
     - bootstrap
+
+- name: setup genesis node
+  hosts: genesis
+  become: true
+  roles:
+    - configure # configure cli
+    - add-accounts
+    - genesis
+  
+- name: configure validator nodes
+  hosts: validators
+  become: true
+  roles:
+    - role: configure
+      type: validator
+
+- name: configure private sentry nodes
+  hosts: private_sentries
+  become: true
+  roles:
+    - role: configure
+      type: private-sentry
+
+- name: configure observer nodes
+  hosts: observers
+  become: true
+  roles:
+    - role: configure
+      type: observer
+
+- name: configure public sentry nodes
+  hosts: public_sentries
+  become: true
+  roles:
+    - role: configure
+      type: public-sentry
+
+- name: configure seed nodes
+  hosts: seeds
+  become: true
+  roles:
+    - role: configure
+      type: seed
+
+- name: copy genesis file to all nodes
+  hosts: all
+  become: true
+  tasks:
+    - name: copy genesis file
+      copy:
+        src: "{{ playbook_dir }}/../persistent_chains/{{ chain_id }}/genesis.json"
```

Comment:
There can be a case when an organization does not own the genesis node. If it has genesis role copies genesis file to that path, so that other nodes can copy it from there.

I believe It is also useful when an organization is joining an existing network.